lib/account/account-manager.js

```diff
@@ -5,6 +5,23 @@
  */
 
 /**
+ * Information about an account holder.
+ *
+ * @typedef {Object} accountManager~accountFile
+ * @property {string} email
+ * @property {string} mfaKey
+ * @property {string} secureHash
+ * @property {string} passwordSalt
+ */
+
+/**
+ *
+ * @typedef {Object} accountManager~challengeSet
+ * @property {string} salt
+ * @property {string} hash
```

Comment:
I don't think we ever send nor receive both the salt and a hash.  I believe this is wrong, but need to chat about it.